index.html

```diff
@@ -21,53 +21,67 @@
-			<div class="flex w-full justify-center md:justify-start pb-24 lg:pb-0 fade-in">
-				<img src="./assets/App-Store.svg" class="h-12 pr-4 bounce-top-icons">
-				<img src="./assets/Play-Store.svg" class="h-12 bounce-top-icons">
-			</div>
-
-		</div>
-		
-		<!--Right Col-->
-		<div class="w-full xl:w-3/5 py-6 overflow-y-hidden">
-			<img class="w-5/6 mx-auto lg:mr-0 slide-in-bottom" src="./assets/devices.svg">
-		</div>
-		
-		
+  <div class="h-screen pb-14 bg-right bg-cover">
+    <!--Nav-->
+    <div class="w-full container mx-auto p-6">
+
+      <div class="w-full flex items-center justify-between">
+        <a class="flex items-center text-red-500 no-underline hover:no-underline font-bold text-2xl lg:text-4xl"
+          href="#">
+          <svg class="fill-current text-gray-500 w-3 h-3 mr-2" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20">
+            <path
+              d="M4 8V6a6 6 0 1 1 12 0v2h1a2 2 0 0 1 2 2v8a2 2 0 0 1-2 2H3a2 2 0 0 1-2-2v-8c0-1.1.9-2 2-2h1zm5 6.73V17h2v-2.27a2 2 0 1 0-2 0zM7 6v2h6V6a3 3 0 0 0-6 0z" />
+            </svg> World Won't Stop!
+        </a>
+
+        <div class="flex w-1/2 justify-end content-center">
+          <a class="inline-block text-blue-300 no-underline hover:text-indigo-800 hover:text-underline text-center h-10 p-2 md:h-auto md:p-4"
+            data-tippy-content="@twitter_handle" href="https://twitter.com/intent/tweet?url=#">
+            <svg class="fill-current h-6" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32">
+              <path
+                d="M30.063 7.313c-.813 1.125-1.75 2.125-2.875 2.938v.75c0 1.563-.188 3.125-.688 4.625a15.088 15.088 0 0 1-2.063 4.438c-.875 1.438-2 2.688-3.25 3.813a15.015 15.015 0 0 1-4.625 2.563c-1.813.688-3.75 1-5.75 1-3.25 0-6.188-.875-8.875-2.625.438.063.875.125 1.375.125 2.688 0 5.063-.875 7.188-2.5-1.25 0-2.375-.375-3.375-1.125s-1.688-1.688-2.063-2.875c.438.063.813.125 1.125.125.5 0 1-.063 1.5-.25-1.313-.25-2.438-.938-3.313-1.938a5.673 5.673 0 0 1-1.313-3.688v-.063c.813.438 1.688.688 2.625.688a5.228 5.228 0 0 1-1.875-2c-.5-.875-.688-1.813-.688-2.75 0-1.063.25-2.063.75-2.938 1.438 1.75 3.188 3.188 5.25 4.25s4.313 1.688 6.688 1.813a5.579 5.579 0 0 1 1.5-5.438c1.125-1.125 2.5-1.688 4.125-1.688s3.063.625 4.188 1.813a11.48 11.48 0 0 0 3.688-1.375c-.438 1.375-1.313 2.438-2.563 3.188 1.125-.125 2.188-.438 3.313-.875z">
+              </path>
+            </svg>
+          </a>
+          <a class="inline-block text-blue-300 no-underline hover:text-indigo-800 hover:text-underline text-center h-10 p-2 md:h-auto md:p-4 "
+            data-tippy-content="#facebook_id" href="https://www.facebook.com/sharer/sharer.php?u=#">
+            <svg class="fill-current h-6" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32">
+              <path d="M19 6h5V0h-5c-3.86 0-7 3.14-7 7v3H8v6h4v16h6V16h5l1-6h-6V7c0-.542.458-1 1-1z"></path>
+            </svg>
+          </a>
+        </div>
+
+      </div>
+
+    </div>
+
```

Comment:
Add closing comments too, gets easier to read where elements end.
Something like <!--Nav Ends-->